### PR TITLE
Update jwt-opaque post to work with the latest Spring Boot 2.4

### PR DESCRIPTION
**Pull request by bdemers**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadeveloper account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL that's approved by Dev Advocacy
- [ ] The content has been ran through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2020-08-07-spring-boot-remote-vs-local-tokens.md [hunk 1]
```diff
@@ -89,7 +89,6 @@ Add two more dependencies to the Maven `pom.xml` file:
 <dependency>
     <groupId>com.nimbusds</groupId>
     <artifactId>oauth2-oidc-sdk</artifactId>
-    <version>7.3</version>
```
Comment (bdemers) [inline _source/_posts/2020-08-07-spring-boot-remote-vs-local-tokens.md:92]: This now defaults to the correct verion

File: _source/_posts/2020-08-07-spring-boot-remote-vs-local-tokens.md [hunk 2]
```diff
@@ -328,6 +327,8 @@ Update your `src/main/resources/application.properties` file to include the prop
 spring.security.oauth2.client.provider.okta.issuer-uri = {yourOktaDomain}/oauth2/default
 spring.security.oauth2.client.registration.okta.client-id = {clientId}
 spring.security.oauth2.client.registration.okta.client-secret = {clientSecret}
+# As of Spring Boot 2.4, you must explicitly scope value
```
Comment (mraible) [inline _source/_posts/2020-08-07-spring-boot-remote-vs-local-tokens.md:330]:
```suggestion
# As of Spring Boot 2.4, you must explicitly set the scope value 
```

File: _source/_posts/2020-08-07-spring-boot-remote-vs-local-tokens.md [hunk 3]
```diff
@@ -412,3 +413,8 @@ Check out these related blog posts to learn more about building secure web appli
 * [Secure Legacy Apps with Spring Cloud Gateway](/blog/2020/01/08/secure-legacy-spring-cloud-gateway)
 
 If you like this blog post and want to see more like it, follow[@oktadev on Twitter](https://twitter.com/oktadev), subscribe to [our YouTube channel](https://youtube.com/c/oktadev), or follow us [on LinkedIn](https://www.linkedin.com/company/oktadev/). As always, please leave a comment below if you have any questions.
+
+<a name="changelog"></a>
+**Changelog**:
+
+* Dec 22, 2020: Updated post to work with Spring Boot 2.4.1 and explicitly set the OAuth scope, as Spring Security no longer provides a default value. (Thanks for the suggestion Wolf!)
```
Comment (mraible) [inline _source/_posts/2020-08-07-spring-boot-remote-vs-local-tokens.md:420]:
```suggestion
* Dec 22, 2020: Updated post to work with Spring Boot 2.4.1 and explicitly set the OAuth scope, as Spring Security no longer provides a default value. Thanks for the suggestion, Wolf! See the code changes in the [example app on GitHub](https://github.com/oktadeveloper/okta-spring-jwt-and-opaque-token-example/commit/d5b63e989f52498eda3f5773a75f3733e0f12b23). Changes to this post can be viewed in [oktadeveloper/okta-blog#508](https://github.com/oktadeveloper/okta-blog/pull/508).
```

**Review by mraible — CHANGES_REQUESTED** [review]
Looks good! Added links so people can see the changes.

**Review by mraible — APPROVED** [review]

**Pull request merged**